addons/hr_timesheet/static/tests/task_with_hours_tests.js

```diff
@@ -0,0 +1,67 @@
+/** @odoo-module */
+
+import { makeView } from "@web/../tests/views/helpers";
+import { click, clickDropdown, editInput, getFixture } from "@web/../tests/helpers/utils";
+
+import { getServerData, updateArch, setupTestEnv } from "./hr_timesheet_common_tests";
+
+
+QUnit.module("hr_timesheet", (hooks) => {
+    let target;
+    let serverData;
+    hooks.beforeEach(async function (assert) {
+        setupTestEnv();
+        serverData = getServerData();
+        updateArch(
+            serverData,
+            { "task_id": "task_with_hours" },
+            { "task_id": "{ 'default_project_id': project_id }" });
```

Comment:
In the case, you applied my suggestion, you could pass an object instead of a string for the context. 🙂 
```suggestion
            { "task_id": { 'default_project_id': project_id } });
```
it is up to you. 😉